ONSdigital/florence
# Allow edit without error for content in current collection

### What

`checkcollectionforuri` check incorrectly gives an error when you click edit in the browse tree for content being edited in the current collection. This fix checks the collection name before
displaying an error.

### How to review

* Create a collection and edit some content
* Go back to the browse tree and click Edit on the page you just edited
* You shouldn't get an error
* Create another collection
* Edit the page you edited in the first collection
* You should get an error telling you the file is in the first collection

### Who can review

@jondewijones @Crispioso
